packages/services/policy/.gitignore

```
@@ -0,0 +1,4 @@
+*.log
+.DS_Store
+node_modules
+dist
```

```suggestion
.DS_Store
```
removed patterns already included in root `.gitignore`, maybe it's better to move `.DS_Store` in root ignore file and remove this one?